tests/client.py

```diff
@@ -35,6 +35,14 @@ def giveup(exc):
                           jitter=None,
                           giveup=giveup,
                           interval=10)
+
+    def get_custom_property(self, url, params=dict()):
+        """Perform a GET using the standard requests method and logs the action"""
+        """Do not handle exception for get failure in custom contact properties"""
+        response = requests.get(url, params=params, headers=self.HEADERS)
```

Comment:
Consider using raise_for_status()

Reply:
This is intentional. I don't want to raise the exception - rather handle it. If the property is already there, I would delete it. See line# 789 where it is called. In the next line, I am checking the status